tools/dep_updaters/update-v8-patch.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -e
+# Shell script to update v8 patch update
+
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
+
+cd "$BASE_DIR"
+
+IS_UP_TO_DATE=$(git node v8 minor | grep "V8 is up-to-date")
```

Comment:
Not sure if `node-core-utils` is installed in the current workflow. We may need a step that installs it.

Reply:
yes we need to install it